app/util/log.ts

```diff
@@ -76,6 +78,32 @@ export function createTextLogger(transports: winston.transport[]): winston.Logge
 const transport = new winston.transports.Console({ level: env.logLevel });
 const logger = process.env.TEXT_LOGGER === 'true' ? createTextLogger([transport]) : createJsonLogger([transport]);
 
+/**
+ * Check if log level is valid.
+ * @param level - the level to check
+ * @throws RequestValidationError - if the log level is not one of winston.config.npm.levels
+ */
+function validateLogLevel(level: string): void {
+  const validLevels = Object.keys(winston.config.npm.levels);
+  if (!validLevels.includes(level)) {
+    throw new RequestValidationError(
+      `Requested to configure log level with invalid level (${level}). Valid levels are: ${listToText(validLevels, Conjunction.AND)}.`);
+  }
+}
+
+/**
+ * Change the log level on the logger transport.
+ * @param level - The new log level that'll be used.
+ * @returns a string indicating the action performed
+ * @throws RequestValidationError - if the log level is not one of winston.config.npm.levels
+ */
+export function configureLogLevel(level: string): string {
+  validateLogLevel(level);
+  const currentLevel = transport.level;
+  transport.level = level;
+  return `Log level was changed from ${currentLevel} to ${level}`;
```

Comment:
I think we should log this message (at either warn or info?) in addition to returning it to the end user. Log it here or in the function that calls this one.